reframe/core/pipeline.py

```diff
@@ -1750,6 +1762,20 @@ class RunOnlyRegressionTest(RegressionTest, special=True):
     module.
     '''
 
+    @_run_hooks()
+    def setup(self, partition, environ, **job_opts):
+        '''The setup stage of the regression test pipeline.
+
+        Similar to the :func:`RegressionTest.setup`, except that no job
+        descriptor is set up for the build of this test.
```

Comment:
```suggestion
        Similar to the :func:`RegressionTest.setup`, except that no build job is created for this test.
```

Reply:
Can you update in a similar way also the documentation of the compile only test?